lib/elementSearch.js

```diff
@@ -97,6 +97,13 @@ async function matchContainsElements(
       translateTo,
       xpathText,
     );
+  if (e === 'button' && (!elements || !elements.length))
```

Comment:
In my opinion we should rename the parameter `e` to `tagName`.

Reply:
I agree we are using variable `e` in many places in this function and in other function calls.
This should be `tagName`, which need to replace everywhere to avoid confusion.

Reply:
I think we should rename it as part of this PR.